gen/genutil.go

```diff
@@ -0,0 +1,126 @@
+package gen
+
+import(
+	"bufio"
+	"bytes"
+	"os"
+   "path/filepath"
```

Comment:
any reason this is not gofmt'ed, or why we would not want to format it?

Reply:
No reason, I overlooked it. I'll fix it